policy/approval/approve.go

```diff
@@ -57,15 +57,19 @@ type RequestReview struct {
 
 func (opts *Options) GetMethods() *common.Methods {
 	methods := opts.Methods
+	defaultBool := true
 	if methods == nil {
-		methods = &common.Methods{
-			Comments: []string{
-				":+1:",
-				"👍",
-			},
-			GithubReview: true,
+		methods = &common.Methods{}
+	}
+	if len(methods.Comments) == 0 {
+		methods.Comments = []string{
+			":+1:",
+			"👍",
```

Comment:
Will this allow a policy to be configured without comment approvals? In other words, will it still be possible to configure a policy where only github reviews (i.e. `github_review=true`) are acceptable forms of approval?

Reply:
Good catch. New commit resolves this.